locale/circuitpython.pot

```diff
@@ -2127,6 +2136,14 @@ msgstr ""
 msgid "UUID value is not str, int or byte buffer"
 msgstr ""
 
+#: ports/raspberrypi/common-hal/memorymap/AddressRange.c
+msgid "Unable to access unaliged IO register"
```

Comment:
Another `make translate` should remove this.

Reply:
Got it.